meetings/2024-09-04.md

```diff
@@ -32,19 +40,92 @@
+* SC: Some of the key spec received updates, even Solid Protocol had a newer version. I can agree with you but about the implementation experience. There were a lot of calls about implementation experience. Considering the number of the active participants to the number of implementers doesn't seem to add up. There is difference what is visible to the CG and what is visible to the outside. Some organizations may be claiming implementing the Solid Protocol but without providing feedback to the CG. I can agree that the community group slowed down but I think do the lack of the implementation feedback. I'm curious how the implementation experience will come up in the WG. Besides that we had a number of updates to the specifications.
+* HZ: I agree with you, there may be other factors like 
+* eP: There is no objective way to measure it, in my subj
+* PAC: I don't have an answer, when Sarven asks if the implementation experience will increase. From my experience some organizations are cautious with technologies which are not on the REC track. 
+* SC: This is a very likely scenario. As far as the communication, there are things being implemented but it isn't communicated in the CG.
+* HZ: This topic was mostly about the announcemtent and celebrating it.
+* HZ: As for implementations we have Manas and ActivityPods which were very active in last year. I think there are implemenations on the solid server side.
+* eP: We have feedback, for example https://forum.solidproject.org/t/solid-interop-in-practice/7701
+* TBL: ... 
+
+
 ### TPAC Demos
 
+* VB (not present, via HackMD): All good. Demos will be published soon.
+* SC: I have more general concern about communication. With respect to https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md#communication . I was approached weather what I say is my own opinion or it represents the CG findings. I think this is important to make sure that you have consent from the group about what you are communicating. There is a difference between something being presented as one's own opinion or more neutral perspective. Weather that will be capture or not, we still need to be catious about that. This is importatn for chairs, for anybody.
+* TBL: I was thinking that we get a bunch of people and we talk about Solid, I try to understand what failure are you worring about. Is it about distinguishing the output of the group over other outputs. Are you worrying that people will present different uses? Or is it about misrepresenting the wishes of the gruop.
+* SC: It is expressing things that are being worked on by the gruop or something build by individuals or organizations. They can say it is Solid but it might not be worked on in the CG. If the work is presented in the way where it is a subset of the work, or captures aspects of the work which aren't belonging. For example excluding some key parts of the specification but include other parts which have nothing to do with it.
```

Comment:
```suggestion
* SC: It is expressing things that are being worked on by the group or something built by individuals or organizations. They can say it is Solid but it might not be worked on in the CG. If the work is presented in the way where it is a subset of the work, or captures aspects of the work which aren't belonging. For example excluding some key parts of the specification when communicating but including other parts which have nothing to do with it.
```

Reply:
```suggestion
* SC: It is about expressing things that are being worked on by the group or something built by individuals or organizations. They can say it is Solid, but it might not be worked on in the CG. If the work is presented in a way where it is a subset of the work, or captures aspects of the work which don't belong. For example, excluding some key parts of the specification when communicating, but including other parts which have nothing to do with it.
```